voluncheer/profiles/models.py

```diff
@@ -219,3 +219,30 @@ def award_volunteer_hours_badges(self):
             self.save()
 
         return hours_remaining
+
+
+def _post_photo_path(instance, filename):
+    return f"opportunities/{instance.volunteer.user.id}/{filename}"
+
+
+class GallaryPost(models.Model):
```

Comment:
Gallary should be spelled "Gallery," here and in the rest of the file.

Reply:
Fixed